satpy/tests/reader_tests/test_ahi_hsd.py

```diff
@@ -78,8 +78,8 @@ def test_region(self, fromfile, np2str):
             self.assertEqual(proj_dict['lon_0'], 140.7)
             self.assertEqual(proj_dict['proj'], 'geos')
             self.assertEqual(proj_dict['units'], 'm')
-            self.assertEqual(area_def.area_extent, (592000.0038256244, 4132000.026701824,
-                                                    1592000.0102878278, 5132000.033164027))
+            self.assertEqual(area_def.area_extent, (592000.0038256242, 4132000.0267018233,
```

Comment:
`numpy.testing.assert_arrays_close` should be more robust here

Reply:
Cahnged.